dot/node.go

```diff
@@ -105,12 +104,6 @@ func InitNode(cfg *Config) error {
 
 	r.Stop()
 
-	// TODO: this should be set in the genesis file, not the config
-	if cfg.Core.BabeThreshold == babe.MaxThreshold {
-		babeCfg.C1 = 1
-		babeCfg.C2 = 1
-	}
-
```

Comment:
sorry you will need to set C1 and C2 here still, can you check if config numerator and denominator are set, and if so set them to `babeCfg.C1` and `babeCfg.C2` respectively?